README.md

```diff
@@ -88,7 +83,7 @@ For browsers not supporting CSS3 3D transforms impress.js adds `impress-not-supp
 class on `#impress` element, so fallback styles can be applied to make all the content accessible.
 
 
-### Even more explanation and technical stuff
+### Even more explanation and technical stuff (and some opinions)
```

Comment:
Why opinions? The text is pretty objective in what one is supposed to expect from impress.js in its current state.